test/e2e/density.go

```diff
@@ -526,15 +500,7 @@ var _ = KubeDescribe("Density", func() {
 				podStartupLatency := PodStartupLatency{Latency: extractLatencyMetrics(e2eLag)}
```

Comment:
should this be `framework.extractLatencyMetrics`?

well, I guess `ExtractLatencyMetrics`, with the function exported.